### ISSUE TITLE
Please, provide `tap` fixture or helper

### ISSUE THREAD

**Issue by unera** [tarantool/luatest#159]
It would be nice to have `tap` helper or fixture.

That can help us to migrate from `taptest` to `luatest`.

example:

```lua
function g.test_foobar()
   tap = g.tap(2, 'Foobar test')   -- plan: 2, name: 'Foobar test'
   tap:ok(bar, 'bar is true')
   tap:eq(baz, baz, 'bar and baz are the same')
end
```

Or:

```lua
function g.test_foobar()
   g.tap.test('Foobar test', function(tap)
       tap:plan(2)
       tap:ok(bar, 'bar is true')
       tap:is(bar, baz, 'bar and baz are the same')
   end)
end
```

Or even:

```lua
function g.taptest_foobar(tap)
    tap:plan(2)
    tap:ok(bar, 'bar is true')
    tap:is(bar, baz, 'bar and baz are the same')
end
```

- plan is optional
- it doesn't need to call `tap:check()` (the call have to be touched after test callback)
- it would be nice to have a way to watch full log with tap log inside tests (luatest option or something else)

**Comment by rosik** [comment]
I'd rephrase the issue: essentially you need a way to rewrite tap tests (in core repo only) with the minimal diff. There're a lot of possibilities to do that. I don't want it to be a part of luatest. No other repos are interested in it. I guess it can be implemented in test helpers, don't make this feature generic.
